### DISCUSSION TITLE
Can I reflect data from before I installed the plugin?

### DISCUSSION THREAD

**Discussion by phono34** [plan-player-analytics/Plan#3371] [Q&A]
Hi, I've been running a server with paper-1.20.2-318 on Ubuntu 22.04 LTS for about 5 months now. 
I recently learned about the Plan plugin, installed it, and roughly finished setting it up, but one thing I didn't like was that when I installed the Plan plugin on a server that was already running, the statistics didn't reflect the data before I installed it.
I was wondering if there is a way to include data from before I installed Plan on the server?
Thank you.

**Comment by AuroraLS3** [comment]
The import system is currently broken. There are several tickets about what you're describing though
